map2grid:
  ros__parameters:
    goal_frame: livox_frame
    map_size: 5.0
    map_resolution: 0.05
    map_topic: map
    publish_topic: global_costmap